controlplane/kubeadm/internal/workload_cluster_conditions.go

```diff
@@ -51,20 +54,37 @@ func (w *Workload) updateExternalEtcdConditions(_ context.Context, controlPlane
 	// When KCP is not responsible for external etcd, we are reporting only health at KCP level.
 	conditions.MarkTrue(controlPlane.KCP, controlplanev1.EtcdClusterHealthyCondition)
 
-	// TODO: check external etcd for alarms an possibly also for member errors
-	// this requires implementing an new type of etcd client generator given that it is not possible to use nodes
-	// as a source for the etcd endpoint address; the address of the external etcd should be available on the kubeadm configuration.
+	// Note: KCP is going to stop setting the `EtcdClusterAvailable` condition to true in case of external etcd.
+	// This will allow tools managing the external etcd instance to use the `EtcdClusterAvailable` to report back status into
```

Comment:
```suggestion
	// Note: KCP is going to stop setting the `EtcdClusterHealthy` condition to true in case of external etcd.
	// This will allow tools managing the external etcd instance to use the `EtcdClusterHealthy` to report back status into
```